README.md

```diff
@@ -73,8 +73,49 @@ git checkout HEAD examples/simple/pi.cc
 
 ### Applying Dredd to a CMake project
 
-TODO(https://github.com/mc-imperial/dredd/issues/28): add instructions
-on how to apply dread to a larger project
+The `examples/project` directory contains a small math library to illustrate how to apply dread to a larger project.
+To compile the project, run the following from the root of the repository:
+```
+cd examples/project
+mkdir build && cd build
+cmake -G Ninja -DCMAKE_EXPORT_COMPILE_COMMANDS=ON ..
+cmake --build .
+```
+You can check the output of the tests before mutations have been applied by running 
+`./mathtest/mathtest` from the build directory.
+
+To apply mutants to the program, either do:
+```
+../../../third_party/clang+llvm-13.0.1/bin/dredd -p compile_commands.js <path-to-file>
+``` 
+to mutate a single file. For example, `../../../third_party/clang+llvm-13.0.1/bin/dredd ../math/src/exp.cc` will apply
+mutants to the file `exp.cc`.
+
```

Comment:
How about here you add something about how they can do git status to see that indeed one file has changed, and git diff to see the effect that Dredd has had - but noting that the changes that Dredd has made are not supposed to be friendly for humans to read.

Reply:
Done